packages/webapp/src/components/ui/OfflineTableNotice/index.tsx

```diff
@@ -0,0 +1,53 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import { wrap } from '~utils/appinsights'
+import { useOffline } from '~hooks/useOffline'
+import { useTranslation } from '~hooks/useTranslation'
+import styles from './index.module.scss'
+import cx from 'classnames'
+
+export const OfflineTableNotice = wrap(function OfflineTableNotice() {
+	const isOffline = useOffline()
+	const { c } = useTranslation()
+
+	const refresh = () => {
+		window.location.reload()
+	}
+
+	return (
+		<>
+			{isOffline ? (
+				<div className={cx(styles.notice)}>
+					<svg
```

Comment:
This SVG can't be added as a `background-image: url()`?

Reply:
Not for anything offline related, react will try to fetch it from the url, but because the user is offline, the svg would not load

Reply:
Okay, maybe this is worth doing a new issue about it, because it's just a problem of pre-fetching.
Also, if it's in the CSS it should be loaded anyway. Unless the styling is applied only if it's offline?